server/shared/database/migrations/20210204115515-hydrate-edited_at-column.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { Comment } = require('../');
+
+exports.up = (_, { Op, col }) => {
+  const attributes = ['id', 'createdAt', 'updatedAt'];
+  const where = { createdAt: { [Op.ne]: col('updated_at') } };
+  return Comment.findAll({ where, attributes })
```

Comment:
Migrations shouldn't use ORM models since models are codebase version specific while migration should be able to run properly only based on the state of the database after the previous migration.